docs/source/platform/client-awareness.md

```diff
@@ -1,4 +1,117 @@
 ---
 title: Identifying clients
-description: How to configure Apollo Engine for client awareness
----
\ No newline at end of file
+description: What is client awareness and how to add it to the Apollo Platform
+---
+
+Client identity is central to the Apollo Platform and enables tracking how all
+the consumers use the data graph. The Apollo Platform allows segmenting usage
```

Comment:
The text here is great! When looking at the Netlify preview, it was a little overwhelming of a wall of text, though, and I think that **bolding** specific pieces would be useful. This sentence in particular is a good one to bold.